httpreq/httpreq.go

```diff
@@ -58,12 +57,33 @@ func New(method, url string, param interface{}) *HttpReq {
 	if err != nil {
 		return &HttpReq{err: err}
 	}
-	req.Header.Set("Content-Type", "application/json")
+	req.Header.Set("Content-Type", dataType)
 	return &HttpReq{
-		req: req,
+		req:      req,
+		dataType: dataType,
 	}
 }
 
+func (r *HttpReq) setDefaultClient() *HttpReq {
```

Comment:
Maybe, we almost use `setDefaultClient()` method.
How about create `defaultClient` variable in `init` method?
Then `setDefaultClient()` method can reuse `defaultClient`.

Reply:
ok,I use init method